src/routes/Sponsors.js

```diff
@@ -2,17 +2,37 @@
 // Email: khangpham@csu.fullerton.edu
 
 import NavBar from '../components/Navbar';
+import '../css/sponsors.css'
 
-
+function Text({name}){
```

Comment:
We don't need the function text here. It's better to move the p tag to where the Text function is called. The reason for this is because we don't expect the text in the h1 tag to change.

Reply:
Fixed I think.

Reply:
Fixed